x-pack/plugins/security_solution/public/management/components/console/components/console_manager/components/console_popup.tsx

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo, PropsWithChildren, ReactNode, useCallback, useMemo, useState } from 'react';
+import {
+  EuiButton,
+  EuiButtonEmpty,
+  EuiIcon,
+  EuiModalBody,
+  EuiModalFooter,
+  EuiModalHeader,
+  EuiModalHeaderTitle,
+} from '@elastic/eui';
+import styled from 'styled-components';
+import { FormattedMessage } from '@kbn/i18n-react';
+import classNames from 'classnames';
+import { ConfirmTerminate } from './confirm_terminate';
+
+const ConsolePopupWrapper = styled.div`
+  position: fixed;
+  top: 100px;
+  right: 0;
+  min-height: 60vh;
+  min-width: 40vw;
```

Comment:
Perhaps use `vmin` if you want the consoles to grow/shrink proportionally to the viewport.

Reply:
No, I don't think the size of the console should change. I will likely be adding explicit width/height to this popup soon so that its fixed.